torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -159,49 +161,66 @@ MemoryLocations AliasDb::getReads(Node* n) const {
   return reads;
 }
 
-static std::string getElementName(const Element* e) {
+std::string AliasDb::getElementName(const Element* e) const {
   if (e->value == nullptr) {
+    // not the most efficient way, but given the fact there are
+    // not too many types and even fewer of them will end up in
+    // wildcardIndex_, we should be fine with a linear search
+    // each time we hit a wildcard leaf
+    for (const auto& ent : wildcardIndex_) {
+      if (ent.second == e) {
+        return std::string("WILDCARD for type ") + typeKindToString(ent.first);
+      }
+    }
     return "WILDCARD";
   } else {
     return e->value->debugName();
   }
 }
 
 void AliasDb::dump() const {
-  std::cout << "\n===1. GRAPH===\n";
-  graph_->dump();
+  std::cout << toString();
+}
+
+std::string AliasDb::toString() const {
+  std::stringstream ss{};
+  std::unordered_map<size_t, Element*> indexToElementMap;
+  // ss << "\n===1. GRAPH===\n";
```

Comment:
Should either remove or uncomment I guess.

Reply:
Uncommented it 😄 Thanks for reviewing it so promptly!